wayat/frontend/lib/services/status/lifecycle_service.dart

```diff
@@ -2,16 +2,18 @@ import 'package:get_it/get_it.dart';
 import 'package:wayat/services/common/api_contract/api_contract.dart';
 import 'package:wayat/services/common/http_provider/http_provider.dart';
 
-class MapStatusService {
+class LifeCycleService {
   final HttpProvider httpProvider = GetIt.I.get<HttpProvider>();
 
   ///Send [POST] request to backend to [Open GoogleMap]
-  Future sendMapOpened() async {
-    await httpProvider.sendPostRequest(APIContract.updateMap, {"open": true});
+  Future notifyMapOpened() async {
```

Comment:
This 2 methods should be merged into one, called notifyLifeCycleState(bool openned) {....}

Reply:
Fix